TimeCalc.java

```diff
@@ -1,5 +1,31 @@
 public class TimeCalc {
     public static void main(String[] args) {
         // Replace this comment with your code
+        String time="";
+        int minutesToAdd=0,hh=0,mm=0;
+        int totalHours=0, totalMinutes=0;
+        int newHours=0, newMinutes=0;
+        time= args[0];

```

Comment:
why not declare:
String time = args[0] ? 
instead of: 
String time="";
time= args[0];